artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -107,6 +108,12 @@
    // Default address drop threshold, applied to address settings with BLOCK policy.  -1 means no threshold enabled.
    public static final long DEFAULT_ADDRESS_REJECT_THRESHOLD = -1;
 
+   public static final DeadLetterAddressRoutingType DEFAULT_DEAD_LETTER_ADDRESS_PREFIX_ROUTING_TYPE = DeadLetterAddressRoutingType.MULTICAST;
+
+   public static final boolean DEFAULT_DEAD_LETTER_ADDRESS_PREFIX_DURABLE = true;
+
+   public static final boolean DEFAULT_DEAD_LETTER_ADDRESS_PREFIX_TEMPORARY = false;
```

Comment:
Parent setting should be the default for the dla

Reply:
E..g this should be three states, true false or original, with original being the default. Same to the others incommented on

Reply:
Added requirement to https://jira.apache.org/jira/browse/ARTEMIS-2420.
I need to close this PR.